tests/models_test.py

```diff
@@ -88,3 +91,28 @@ def test_no_top_variable_shape_4(app, last_feature_dim):
     input_shape = (None, None, 4)
     model = app(weights=None, include_top=False, input_shape=input_shape)
     assert model.output_shape == (None, None, None, last_feature_dim)
+
+
+@pytest.mark.parametrize("command_name", cli.commands.keys())
+def test_compare_model_summary(command_name: str, snapshot):
+    # FIXME: ideally, we'd directly call the relevant command here rather than
```

Comment:
I agree, we definitely should just call the models directly here.

Reply:
If we only want the model summaries it'd be as easy as 
```python
@parametrize
def test_model_summary(app, last_feature_dim, capsys, snapshot):
    input_tensor = keras.layers.Input(shape=(224, 224, 3))
    model = app(weights=None, input_tensor=input_tensor)
    lq.models.summary(model)
    out, err = capsys.readouterr()
    snapshot.assert_match(out)
```

But I think it would be nice to actually compile all experiments

Reply:
All the models do get compiled in other tests so I think it's okay to not do that here.

Reply:
Yeah I don't mean just the models, I mean actually running the experiment (with a dry run). As far as I can see that currently doesn't happen. Though it might be worth separating the model summary snapshot and the compilation test into two separate tests

Reply:
Yeah I think separating the two is fine, but also with Larq Zoo the training loops are definitely less of a priority than the models + weights.